### DISCUSSION TITLE
Date Partition for future dates

### DISCUSSION THREAD

**Discussion by shionw** [dagster-io/dagster#19040] [Q&A] [answered]
Is there any way to configure the date partitions(specifically either `MonthlyPartitionsDefinition ` or `TimeWindowPartitionsDefinition`) for future dates? Looks like the default is set to the last full month and I want to configure for the current month and so on.

The existing docs/library does not?  seem to describe a way to configure it so wanted to ask here. 

Thanks in advance!

**Comment by the4thamigo-uk** [comment, marked as answer]
I think this is maybe something like what you are looking for? 


> end_offset (int) – Extends the partition set by a number of partitions equal to the value passed. If end_offset is 0 (the default), the last partition ends before the current time. If end_offset is 1, the second-to-last partition ends before the current time, and so on.

https://docs.dagster.io/_apidocs/partitions#dagster.MonthlyPartitionsDefinition

**Reply by the4thamigo-uk** [reply]
I often set this to `1` to include the 'current' partition

**Reply by shionw** [reply]
that was it! thanks for the guidance.
